docs/users-guide/04-asking-questions.md

```diff
@@ -105,18 +105,30 @@ Once you're done setting your metrics and groupings, click Done to close the Sum
 
 If you want to jump ahead and learn about [how to change the visualization](05-visualizing-results.md) of your results, by all means, feel free.
 
-### Viewing an individual record's details
+## Viewing an individual record's details
 
 Click on a record's ID number (or primary key) to see more information about a given user, order, venue, etc. You can see all fields related to that one record and all connected tables that are hidden in the table view for the sake of readability. Press the right or left arrow keys, or click on the arrows to the right or left of the screen to page through the other records in the current list.
 
 ![Record details](./images/notebook/record-details.png)
 
-### Downloading Your Results
+## Downloading Your Results
 
 You can download or export the results of a question by clicking on the Download arrow in the lower right of the screen. Results can be downloaded into .csv, .xlsx, or .json files. The maximum download size is 1 million rows.
 
 ![Download Button](./images/download-button.png)
 
+## Starting new explorations from saved questions
+
+Every time you start modifying a saved question, Metabase will start a new question for you, with a placeholder title, and let you know which question you started from. 
+
+![Starting from a saved question](./images/notebook/started-from-saved-question.png)
+
+So feel free to play around with any existing question, as you won't have any effect on the existing question. When you hit **Save** on the question, you can choose either to save as a new question (the default), or you can overwrite the existing question you started from.
```

Comment:
Would be great if we could find a replacement for the duplication of "existing" here.